common/router_data_model.go

```diff
@@ -100,6 +350,11 @@ func (self *Identity) Equals(other *Identity) bool {
 		return false
 	}
 
+	if len(self.ServiceConfigs) != len(other.ServiceConfigs) {
+		log.Info("identity updated, number of service configs changed")
```

Comment:
Should this be a debug or a trace? I don't know the frequency of this output, but I figure it might be high.

Also another on line 377.

Reply:
fixed